src/util/positioning.ts

```diff
@@ -112,6 +113,8 @@ export function getPopperOptions({placement, baseClass}: PositioningOptions): Pa
       bsModifier,
       flip,
       preventOverflow,
+      offset,
+      {name: 'offset', options: {offset: [0, 2]}},
```

Comment:
but you've added it for all components, not only to the popover...
tooltips are now not positioned correctly, for example

Reply:
Oh yes, you're right @maxokorokov, sorry, I should have known it, as it's already done for the popover. 
New pushed done.